fiberplane/fiberplane
# Show responses from proxied requests in requestor, even if request is not to linked service

Summary:

- Render response AS-IS (when we can) for requests made through requestor
  - For binary responses, we only handle images right now
  
- Allow requests to services other than the target service for FPX
  - This required A TON of changes since it broke a fundamental assumption of the UI, that you're only testing your service

- Simplify `RequestorInput` to be fully controlled by the `path` state inside the main Requestor reducer
  - This required reconfiguring how we automagically add the service baseUrl to requests, as well as how we match history entries to the form input fields

- Remove the `Headers` tab from the response panel and put headers as a collapsed bit above the response body

***

To test:

- Go through your usual flow, make sure nothing broke
- Make requests to a non-monitored service, and make sure the responses render as expected
- Try loading entries from history
- Try filling in request parameters, generating a prompt for testing, etc
